tests/integration-tests/tests/cloudwatch_logging/cloudwatch_logging_boto3_utils.py

```diff
@@ -0,0 +1,88 @@
+import json
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+
+import utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_log_groups():
+    """Get list of log groups."""
+    logs_client = boto3.client("logs")
+    log_groups = logs_client.describe_log_groups().get("logGroups")
```

Comment:
minor: we can avoid to define the logs_client
```python
log_groups = boto3.client("logs").describe_log_groups().get("logGroups")
```

Reply:
Done.